R/net.accessor.R

```diff
@@ -438,6 +438,35 @@ get_control <- function(dat, item, override.null.error = FALSE) {
   return(out)
 }
 
+#' @rdname net-accessor
+#' @param network index of network for which to get control
+#' @export
+get_network_control <- function(dat, item, override.null.error = FALSE, network) {
+  if (missing(network)) {
+    stop("`get_network_control` requires `network` argument.")
+  }
```

Comment:
`network` being a mandatory argument, we should not handle it with `missing` and instead let R crash in the `normal` way if it's not provided.

Also, it is usually agreed that `missing` should be avoid in general and replaced with a default value. Usually, `network = NULL` in the function signature and `if (is.null(network))` makes it immediately visible that the `network` argument is optional without the need to read the body of the function or the documentation.

Reply:
I don't think R's default handling of a missing argument used as an index in a `list` is really what we want here.  For example, if we define

```
f <- function(a,b) a[[b]]
```

then

```
f(a = list(1,2,3))
```

evaluates to `NULL`; there is no error thrown, even though `b` is missing and is used to index `a`.

We can restore an error message without an explicit `missing` check by calling `force(b)` before returning `a[[b]]`.

Reply:
the `force` mechanism is used in dbdaab3972fab3d4a912d9ed89b844ebb50e0080 but I will leave this open for now in case there is more to discuss

Reply:
Oh I did not know this case! Thanks.

In this case I can see that `missing` is actually the right call, sorry for this comment